apps/nextjs/package.json

```diff
@@ -17,19 +17,18 @@
     "@thunderstore/cyberstorm": "^0.1.0",
     "@thunderstore/dapper": "^0.1.0",
     "@thunderstore/hooks": "^0.1.0",
-    "@types/react": "^17.0.38",
+    "@types/react": "18.0.35",
```

Comment:
Could it maybe be enough to update React versions of `apps/cyberstorm-nextjs` and `packages/cyberstorm` (and perhaps `apps/cyberstorm-storybook` to match? Not that I'm against updating the others, but seems like here they are just updated without any proper testing or thought?

Reply:
It's probably best to keep them in sync if we can help it, and I'm pretty sure that's also a requirement on some level given the cross-package dependencies.

Reply:
The problem was that I couldn't have multiple versions of these packages exist at the same time. Or at least not with such a big version difference (17vs18) or it wouldn't work. I can try if it works with just what @anttimaki mentioned 🤷

Reply:
> I can try if it works with just what @anttimaki mentioned

no need to, better to keep them in sync as mentioned